front/app/components/ConsentManager/ConsentManager.test.tsx

```diff
@@ -7,7 +7,7 @@ import eventEmitter from 'utils/eventEmitter';
 
 // utils
 import { registerDestination } from './destinations';
-import { isAdmin, isModerator } from 'services/permissions/roles';
+import { isAdmin, isNotRegularUser } from 'services/permissions/roles';
```

Comment:
This was the original isModerator, which only checks if a user is "more" than a normal user.

Reply:
They are not necessarily, they could also be an admin. The implementation of this function check if highest role != user.

Reply:
FYI, I'm not a fan of "isNot" in variable/function names but making it 'isRegularUser', would have required me to add ! in dozens of places. But I could still do that if you think that that would be clearer.

Reply:
In cases where we need both admins and moderators, we could then use `isAdmin` and `isModerator` or create one that adds both if there are many such usages

Reply:
Yes, or we would use a canModerate function to which we pass the user. For these isModerator, isAdmin, isRegularUser, I expect exact role checks.

So here, what I can do is change it to isRegularUser and then add a ! everywhere. Then I add a ticket for this in platform. Wdyt?

Reply:
Sounds good